.gitignore

```diff
@@ -1 +1,22 @@
+.clangd/
```

Comment:
Sue me

Reply:
OBJECTION

Please undo your changes to this file.

Reply:
Sustained 👨🏻‍⚖️